mabl snippets/svgAttributeExtraction.js

```diff
@@ -10,10 +10,17 @@ function mablJavaScriptStep(mablInputs, callback) {
     // Can be used to target a div element by class name, id, etc. Replace element with your desired element.
     const element = document.querySelector('.element');
     
+    if (element) {
```

Comment:
Consider using the "guard condition" pattern.

```
if (!element) {
  throw(...)
}

// Do the rest
```

It minimizes nested scopes and makes the code a bit easier to read and reason about.